oslabs-beta/KataLog
# Metrics Bar Graph

##Overview

Issue Type

[] Bug
[x]Feature
[]Tech debt
Description
Added in bar graph into dashboard page. In the codebase: added two components metrics.tsx and BarGraph.tsx. 

Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix
Navigate to dashboard page and choose a project with logs. 

Previous Behavior
No graph =(

Expected Behavior
Bar graph should update with warn, error, info, and ok bars. 

How Has This Been Tested
functionally tested

Screenshots & Videos
N/A.

Additional Context
N/A.
